README.md

```diff
@@ -7,6 +7,7 @@ Requirements:
 - XeLaTeX
 - latexmk
 - Fira fonts
+- "pygmentize" package
```

Comment:
Ah, I wasn't aware that in addition to the `pygments` library the `pygmentize` script is also required.